core/services/ocr2/plugins/ocr2keeper/evm21/upkeepstate/store.go

```diff
@@ -76,19 +100,23 @@ func (u *upkeepStateStore) Start(pctx context.Context) error {
 
 	u.lggr.Debug("Starting upkeep state store")
 
-	ticker := time.NewTicker(GCInterval)
-	defer ticker.Stop()
+	timer := time.NewTimer(u.cleanCadence)
```

Comment:
revert to ticker

Reply:
Why?

Reply:
Why timer?
We need ticker functionality, no need to change to timer